packages/nx-python/package.json

```diff
@@ -16,15 +16,13 @@
     "chalk": "^4.1.1",
     "fs-extra": "^11.1.0",
     "@iarna/toml": "^2.2.5",
-    "uuid": "^9.0.0",
-    "file-uri-to-path": "^2.0.0",
     "cross-spawn": "^7.0.3",
     "command-exists": "^1.2.9",
     "lodash": "^4.17.21",
     "@nx/devkit": "^20.0.0",
-    "nx": "20.1.2",
     "ora": "5.3.0",
-    "semver": "^7.5.3"
+    "semver": "^7.5.3",
+    "nx": "20.1.2"
```

Comment:
Hey @FrozenPandaz, thanks for raising this PR.

The nx dependency still in the package.json, and I also noticed you are removing some other packages.

That doesn't feel right, could you check pls?

Reply:
Oh no, I'm so sorry. Funnily enough, the nx dependency-check lint rule acted up for me locally. I'll take a look at that but it's not related to this PR.

I ignored the imports from the lint rule... It's still okay for you to import nx without depending on it because you have a peer dependency.. it should be resolveable. Ideally our lint rule wouldn't warn about it. Also ideally, you should not import directly from `nx` but we recognize that there are some private APIs which you are using. We'll take note of it and see if we can expose those as `@nx/devkit` utils for you to move to.